### ISSUE TITLE
Add more details to navigation rotation widget 

### ISSUE THREAD

**Issue by AnastasiaTiutina** [osmandapp/OsmAnd#10093]
Add more details to the navigation rotation widget 
Perhaps it should be an intersection under the  arrow
![1](https://user-images.githubusercontent.com/72389175/97412328-15244d00-190a-11eb-94ef-e6a97c6b4bfc.jpg)

**Comment by scaidermern** [comment]
Interesting idea. Is it possible to visualize the intersection, similar to roundabouts?

**Comment by vshcherb** [comment]
Yes, that what we need to do i.e. reflect real look of roads intersection. Probably it won't fit all people need but it's definitely could be a good option to visualize upcoming turn
